### PR TITLE
Add documentation for vendor_ccs.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: docs/getting-started/new_project_guide.md [hunk 1]
```diff
@@ -87,6 +88,14 @@ You project's homepage.
 The primary contact and list of other contacts to be CCed. Each person listed gets access to ClusterFuzz, including crash reports and fuzzer statistics, and are auto-cced on new bugs filed in the OSS-Fuzz
 tracker. If you're a primary or a CC, you'll need to use a [Google account](https://support.google.com/accounts/answer/176347?hl=en) to get full access. ([why?]({{ site.baseurl }}/faq/#why-do-you-require-a-google-account-for-authentication)).
 
+### vendor_ccs (optional) {#vendor}
+The list of vendor email addresses that are downstream consumers of the project and want access to
+the bug reports as they are filed.
+
+Any changes to this list must follow these rules:
+- Should be approved by the project maintainer (GitHub PR, project mailing list, etc).
```
Comment (Dor1s) [inline docs/getting-started/new_project_guide.md]: nit: maybe "(GitHub PR, project mailing list, etc)." -> "by leaving a comment on the pull request or replying on a project mailing list)" to be more concrete?

Comment (Dor1s) [reply docs/getting-started/new_project_guide.md]: I'd also drop "Should be", as we've just said "must follow" above

File: docs/getting-started/new_project_guide.md [hunk 2]
```diff
@@ -87,6 +88,14 @@ You project's homepage.
...
+- Should be an organization email address.
```
Comment (Dor1s) [inline docs/getting-started/new_project_guide.md]: suggestion: "An organization email address is used."

File: docs/getting-started/new_project_guide.md (hunk 1 above)
Comment (inferno-chromium) [reply docs/getting-started/new_project_guide.md]: Done!

File: docs/getting-started/new_project_guide.md (hunk 2 above)
Comment (inferno-chromium) [reply docs/getting-started/new_project_guide.md]: Done!

**Pull request merged**